index.js

```diff
@@ -60,7 +64,11 @@ function config(opts) {
             cfg[name] = value;
         }
         else if ( opt.type === 'boolean' ) {
-            booleanValue = valid.boolean[value.toLowerCase()];
+            if ( typeOf(value) != 'boolean' ) {
```

Comment:
I've introduced a new function as well, which should be better than typeof. Please see: http://stackoverflow.com/a/28475765/806544

Also, please not, I'm not a developer. So feel free to just ignore the pull request and update the example configuration. Because in the example config you are defining valid booleans already instead of strings, meaning the toLowerCase() will break on a boolean.